chaincode/testtuple.go

```diff
@@ -130,8 +130,7 @@ func (testtuple *Testtuple) SetFromTraintuple(db LedgerDB, traintupleKey string)
 		if err != nil {
 			return errors.BadRequest(err, "could not retrieve composite traintuple with key %s", traintupleKey)
 		}
-		// TODO (ask Camille?): are those the right permissions?
-		permissions = compositeTraintuple.OutHeadModel.Permissions
+		permissions = MergePermissions(compositeTraintuple.OutHeadModel.Permissions, compositeTraintuple.OutTrunkModel.Permissions)
```

Comment:
I think the merge is not necessary in this case. 
OutHeadModel is forced to one node, so the intersection with one node will always generate the same node.
I think this is why it was forced to compositeTraintuple.OutHeadModel.Permissions

Reply:
Fixed https://github.com/SubstraFoundation/substra-chaincode/pull/5/commits/45748822a3926f04b24151bfd0d66dd8144ea9d1